gnd/cosmos/install_cosmos_plugin.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+#
+# Copy COSMOS plugin configuration to argument directory
+# This will likely be moved to the Adamant repository
+# COSMOS install directory argument should be relative to the Linux example path
+# Ex. `./install_cosmos_plugin.sh linux_example cosmos-project` if COSMOS is adjacent to adamant_example
+
+adamant_assembly_name=$1
+cosmos_install_name=$2
```

Comment:
Add some error handling to exit out with message if either of these variables are not provided by the caller.

Reply:
Addressed in [a6d5741](https://github.com/lasp/adamant_example/pull/18/commits/a6d57412be10bf4aef6a0472f17ff5fa5381189a).